Classes/Command/ProjectionCommandController.php

```diff
@@ -116,11 +116,16 @@ public function replayCommand($projection, $quiet = false)
             $this->output->progressStart();
         }
         $eventsCount = 0;
-        $this->projectionManager->replay($projectionDto->getIdentifier(), function () use (&$eventsCount, $quiet) {
+        $startOfLastThousandEvents = microtime(true);
+        $this->projectionManager->replay($projectionDto->getIdentifier(), function () use (&$eventsCount, $quiet, &$startOfLastThousandEvents) {
             $eventsCount ++;
             if (!$quiet) {
                 $this->output->progressAdvance();
             }
+            if ($eventsCount % 1000 === 0) {
+                $this->outputLine('Time taken for 1000 events: ' . (microtime(true) - $startOfLastThousandEvents));
+                $startOfLastThousandEvents = microtime(true);
```

Comment:
yes.

I am thinking about making the output configurable (e.g. "display progress by default" vs "display detailed progress"). WDYT?